modules/apps/app-builder/app-builder-lang/src/main/resources/content/Language.properties

```diff
@@ -48,6 +48,7 @@ filter-entries=Filter Entries
 filter-entries-by-columns=Filter Entries by Columns
 filter-entries-by-setting-a-value-for-this-column=Filter entries by setting a value for this column.
 filter-entries-by-setting-filters-for-each-column=Filter entries by setting filters for each column.
+for-all-forms-of-this-object=For all forms of This object
```

Comment:
Can we use uncapitalized  T in the word "This"? 

And also it possibel to split this commit in two? One for the creation of the key and the other for buildLang